src/site/content/en/blog/multi-origin-pwas/index.md

```diff
@@ -121,6 +112,10 @@ In most cases, there's no solution for this, but a workaround can be applied for
 The previous technique can help mitigating the potential UI change in a small part of the site, where the user can perform an action in a subdomain and return to the main origin (like in a login flow), but won't be an efficient technique to implement for entire paths, including many pages hosted in subdomains (like entire site sections).
 {% endAside %}
 
+{% Aside %}
+In the context of [Trusted Web Actitivies](https://developers.google.com/web/updates/2019/02/using-twa), there's a recommended way of avoiding this issue, by validating all origins using Digital Asset Links. You can find more information [here](https://github.com/GoogleChrome/android-browser-helper/tree/master/demos/twa-multi-domain/).
```

Comment:
Please rephrase the second link's text by following these guidelines: https://developers.google.com/style/link-text

Reply:
Perfect. Good job again @demianrenzulli.